### PR TITLE
Replaced checking for null with operator let

### PR DESCRIPTION
**Pull request by GeorgCantor**

### CODE REVIEW

**Comment by GeorgCantor** [comment]
I signed it!

пн, 31 авг. 2020 г. в 19:35, googlebot <notifications@github.com>:

File: common/src/main/java/com/example/android/uamp/media/MusicService.kt [hunk 1]
```diff
@@ -478,11 +478,11 @@ open class MusicService : MediaBrowserServiceCompat() {
 
         override fun onPrepare(playWhenReady: Boolean) {
             val recentSong = storage.loadRecentSong()
```
Comment (nic0lette) [inline common/src/main/java/com/example/android/uamp/media/MusicService.kt:480]: We could probably make this even easier by changing this line to: `val recentSong = storage.loadRecentSong() ?: return`, then we wouldn't need the null check at all.

Comment (GeorgCantor) [reply common/src/main/java/com/example/android/uamp/media/MusicService.kt:480]: OK

**Comment by nic0lette** [comment]
Thank you :)

**Pull request merged**